regen/opcodes

```diff
@@ -261,7 +261,7 @@ list		list			ck_null		m@	L
 lslice		list slice		ck_null		2	H L L
 anonlist	anonymous array ([])	ck_fun		ms@	L
 anonhash	anonymous hash ({})	ck_fun		ms@	L
-emptyavhv	empty anon hash/array	ck_fun		sT@	L
+emptyavhv	empty anon hash/array	ck_fun		sT0	L
```

Comment:
Does this still take a list argument?

Though I'm not sure it matters anywhere.

Reply:
I'll remove the argument.